lib/loader/index.js

```diff
@@ -38,23 +38,29 @@ const ARRAY_SIZE = 16;
 
 const BIGINT = typeof BigUint64Array !== "undefined";
 const THIS = Symbol();
-const CHUNKSIZE = 1024;
+
+const STRING_DECODE_THRESHOLD = 32;
+
+let decoder;
+if (typeof process !== 'undefined') {
+  // node.js
+  decoder = new (require('util').TextDecoder)('utf-16le');
```

Comment:
This looks problematic, in that a user including the loader in a bundle and naturally not being aware of this dependency might get multiple megabytes of charset definitions they don't need.

Reply:
Since node.js 11 we don't need this. So if we don't care about older versions we could remove this